core/src/main/java/tc/oc/pgm/spawner/objects/SpawnablePotion.java

```diff
@@ -0,0 +1,44 @@
+package tc.oc.pgm.spawner.objects;
+
+import java.util.List;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.entity.ThrownPotion;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.PotionMeta;
+import org.bukkit.metadata.FixedMetadataValue;
+import org.bukkit.potion.PotionEffect;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.spawner.Spawnable;
+import tc.oc.pgm.spawner.Spawner;
+
+public class SpawnablePotion implements Spawnable {
+  private ItemStack potionItem;
+  private String METADATA_VALUE;
```

Comment:
```suggestion
  private String metadataValue;
```

Reply:
`METADATA_VALUE` doesn't change, and is also capitalized in SpawnableItem, perhaps we should make both the metadata and ItemStack as a final?

Reply:
Even when its `final` its should still be camelCase. Only static final things should be UPPER_CASE.